### PR TITLE
DPG/AOTrack:Make Axis configurables

### PR DESCRIPTION
**Pull request by VDiBella**

### CODE REVIEW

File: DPG/Tasks/AOTTrack/tagAndProbeDmesons.cxx [hunk 1]
```diff
@@ -1149,13 +1149,15 @@ struct ProbeThirdTrack {
...
 
-    const AxisSpec axisPtProbe{{0.05f, 0.1f, 0.2f, 0.3f, 0.4f, 0.5f, 0.6f, 0.7f, 0.8f, 0.9f, 1.0f, 1.2f, 1.5f, 2.0f, 2.5f, 3.0f, 3.5f, 4.0f, 4.5f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f, 10.f, 12.f, 15.f, 20.f, 25.f, 30.f}};
-    const AxisSpec axisPtTag{{0.05f, 0.1f, 0.2f, 0.3f, 0.4f, 0.5f, 0.6f, 0.7f, 0.8f, 0.9f, 1.0f, 1.2f, 1.5f, 2.0f, 2.5f, 3.0f, 3.5f, 4.0f, 4.5f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f, 10.f, 12.f, 15.f, 20.f, 25.f, 30.f}};
-    const AxisSpec axisPtD{{0.f, 0.5f, 1.f, 1.5f, 2.0f, 2.5f, 3.0f, 3.5f, 4.0f, 4.5f, 5.0f, 5.5f, 6.0f, 6.5f, 7.0f, 7.5f, 8.0f, 8.5f, 9.0f, 9.5f, 10.f, 11.f, 12.f, 14.f, 16.f, 20.f, 24.f, 36.f, 50.f}};
-    const AxisSpec axisEtaProbe{20, -1.f, 1.f};
-    const AxisSpec axisNumCrossRowTpc{51, 49.5f, 100.5f};
-    const AxisSpec axisTpcChi2PerClus{8, 2.f, 10.f};
-    const AxisSpec axisNumCluIts{5, 2.5f, 7.5f};
+
+    Configurable<int> selectionFlag{"selectionFlag", 32, "Selection Flag for the considered charm hadron"};
```
Comment (mfaggin) [inline DPG/Tasks/AOTTrack/tagAndProbeDmesons.cxx:1153]: you can remove it, since you do not use it anywhere

**Review by mfaggin — CHANGES_REQUESTED** [review]
thanks @VDiBella . On top of the small comment at line 1153, please fix the clang format. Here instructions:

 - type `alienv enter Clang/latest`
 - `clang-format -style=file -i path-to-your-file`

More info here, in case of need: https://github.com/AliceO2Group/CodingGuidelines

**Comment by vkucera** [comment]
> thanks @VDiBella . On top of the small comment at line 1153, please fix the clang format. Here instructions:
> 
>     * type `alienv enter Clang/latest`
> 
>     * `clang-format -style=file -i path-to-your-file`
> 
> 
> More info here, in case of need: https://github.com/AliceO2Group/CodingGuidelines

@mfaggin There is no need to run `clang-format`. There is the automatic formatting PR for that.

**Comment by vkucera** [comment]
@VDiBella It seems that the updates from upstream did not get merged into your master on top of the commit [5c924b2](https://github.com/AliceO2Group/O2Physics/pull/7132/commits/5c924b2032294f94d31b89d36c19b2b2c48c53f6) cleanly so now the PR appears to contain those changes as well. You can try to rebase on top of the upstream master to fix it.

**Review by mfaggin — DISMISSED** [review]

**Review by mfaggin — APPROVED** [review]

**Pull request merged**